docs/source/examples/managed-jobs.rst

```diff
@@ -958,31 +958,59 @@ Best practices for scaling up the jobs controller
 
 The number of active jobs that the controller supports is based on the controller size. There are two limits that apply:
 
-- **Actively launching job count**: maxes out at ``4 * vCPU count``.
+- **Actively launching job count**: maxes out at ``8 * floor((memory - 2GiB) / 3.59GiB)``.
```

Comment:
no need to be in this PR: The fomrula is good, but it would be easier to understand with a table like the following
| Controller memory | Max actively launching jobs | Max running jobs |
|:-------------------:|:----:|:--:|
| 16 (default) | 24 | 610 |
| 32 | ... | ... |